pennylane/resource/resource.py

```diff
@@ -108,26 +111,26 @@ def resources(self) -> Resources:
         ...     def resources(self):
         ...         return Resources(num_wires=self.num_wires, num_gates=3, depth=2)
         ...
-        >>> op = CustomOp()
+        >>> op = CustomOp(wires=[0, 1])
         >>> print(op.resources())
         wires: 2
         gates: 3
         depth: 2
-        shots: 0
+        shots: Shots(total=None)
         gate_types:
         {}
         gate_sizes:
         {}
         """
 
 
-def _count_resources(tape, shots: int) -> Resources:
+def _count_resources(tape, shots: Shots) -> Resources:
```

Comment:
I'll remove the `shots` argument in my PR since they're already on the tape.

Reply:
Awesome! Thanks